rocketpy/simulation/flight.py

```diff
@@ -3045,17 +2812,40 @@ def _calculate_pressure_signal(self):
                 "Pressure - Without Noise (Pa)",
                 "linear",
             )
-            parachute.noisy_pressure_signal_function = Function(
-                parachute.noisy_pressure_signal,
-                "Time (s)",
-                "Pressure - With Noise (Pa)",
-                "linear",
-            )
             parachute.noise_signal_function = Function(
                 parachute.noise_signal, "Time (s)", "Pressure Noise (Pa)", "linear"
             )
+            parachute.noisy_pressure_signal_function = (
+                parachute.clean_pressure_signal_function
+                + parachute.noise_signal_function
+            )
 
-        return None
+    @cached_property
+    def __evaluate_post_process(self):
+        """Evaluate all post-processing variables by running the simulation
+        again but with the post-processing flag set to True.
+
+        Returns
+        -------
+        np.array
+            An array containing all post-processed variables evaluated at each
+            time step. Each element of the array is a list containing:
+            [t, ax, ay, az, alpha1, alpha2, alpha3, R1, R2, R3, M1, M2, M3]
+        """
+        self.__post_processed_variables = []
+        for phase_index, phase in self.time_iterator(self.flight_phases):
+            init_time = phase.t
+            final_time = self.flight_phases[phase_index + 1].t
+            current_derivative = phase.derivative
+            for callback in phase.callbacks:
+                callback(self)
+            for step in self.solution:
+                if init_time < step[0] <= final_time or (
+                    init_time == self.t_initial and step[0] == self.t_initial
+                ):
+                    current_derivative(step[0], step[1:], post_processing=True)
+
+        return np.array(self.__post_processed_variables)
```

Comment:
This gives wrong values for the accelerations of the rail phase.
In the old code there used to be two `retrieve_arrays` methods because the accelerations were treated differently from the forces/moments

I would suggest merging this like this though, since its pretty easy to fix with the code on #586

Reply:
Fixing this probably makes https://github.com/RocketPy-Team/RocketPy/pull/581/commits/0eb4ae8b9c0779d232b126de58f8394771ff3435 reversable

Reply:
And also https://github.com/RocketPy-Team/RocketPy/commit/79be71439d1f43f9d6b2a5e20a9a8eef701807d9

Reply:
I understand the problem now.

I can agree with a roll back here, but I wanna state that I don't think this behavior is good enough: instead of calling u_dot_rail1 with post_process=True, we are calling the u_dot_generalized with post_process=True. 

In the future I would like to see an enhancement here.

Reply:
Fixed here: [92e93a7](https://github.com/RocketPy-Team/RocketPy/pull/581/commits/92e93a7f616f9c7a02b281fba09418e96f387b61)